spec/lib/tasks/bulk_email_spec.rb

```diff
@@ -1,4 +1,46 @@
 RSpec.describe "bulk_email" do
+  describe "brexit_subscribers" do
+    let(:tags) { { brexit_checklist_criteria: { any: %w[visiting-eu] } } }
+    let!(:list) { create(:subscriber_list, tags: tags) }
+    let(:task) { Rake::Task["bulk_email:brexit_subscribers"].invoke }
```

Comment:
Do we need this? We seem to be [invoking it manually later anyway](https://github.com/alphagov/email-alert-api/pull/1384/files#diff-a7e3b6edd154b4653f447b46ea4c18feR25).

Reply:
I've removed it thanks